rr_custom_msgs/msg/DoseRate.msg

```diff
@@ -0,0 +1,3 @@
+Header header
+int32 dose_stamp
```

Comment:
what is the dose_stamp?

Reply:
It is the data coming from the radiation sensor with its own time stamp (quote @pepeRossRobotics )

Reply:
Is the type OK here? Usually timestamps are of type uint64 wherever possible, it might be worth double checking this to avoid overflows.